### PR TITLE
Add JAS integration tests

### PR DESCRIPTION
**Pull request by omerzi**
- [x] All [tests](https://github.com/jfrog/jfrog-cli/CONTRIBUTING.md#tests) have passed. If this feature is not already covered by the tests, new tests have been added.
- [x] The pull request is targeting the `dev` branch.
- [x] The code has been validated to compile successfully by running `go vet ./...`.
- [x] The code has been formatted properly using `go fmt ./...`.

---
Depends on:
https://github.com/jfrog/jfrog-cli-core/pull/884

### CODE REVIEW

File: xray_test.go [hunk 1]
```diff
@@ -660,6 +666,53 @@ func TestXrayOfflineDBSyncV3(t *testing.T) {
...
+	initXrayTest(t, commands.GraphScanMinXrayVersion)
+	tempDirPath, createTempDirCallback := coretests.CreateTempDirWithCallbackAndAssert(t)
+	defer createTempDirCallback()
+	projectDir := filepath.Join(filepath.FromSlash(tests.GetTestResourcesPath()), filepath.Join("xray", project))
+	// Copy the multi project from the testdata to a temp dir
+	assert.NoError(t, fileutils.CopyDir(projectDir, tempDirPath, true, nil))
+	// Configure a new server named "default"
+	createJfrogHomeConfig(t, true)
+	defer cleanTestsHomeEnv()
+	baseWd := changeWD(t, tempDirPath)
```
Comment (sverdlov93) [inline xray_test.go:690]: chdirCallback := clientTestUtils.ChangeDirWithCallback(t, wd, tempDirPath)
	defer chdirCallback()

File: xray_test.go [hunk 2]
```diff
@@ -660,6 +666,53 @@ func TestXrayOfflineDBSyncV3(t *testing.T) {
...
+	defer func() {
+		assert.NoError(t, os.Chdir(baseWd))
+	}()
+	return xrayCli.WithoutCredentials().RunCliCmdWithOutput(t, "audit", "--format="+format)
+}
+
+func verifySimpleJsonJasResults(t *testing.T, content string, minIacViolations, minSecrets, minApplicable, minNotApplicable int) {
```
Comment (sverdlov93) [inline xray_test.go:697]: can we also test the ignore rule?

**Review by sverdlov93 — COMMENTED** [review]
amezing

File: xray_test.go (hunk 2 above)
Comment (omerzi) [reply xray_test.go:697]: Already tested internally. One of the JSON test files has a suppression, so I verified that we have one less result.

**Pull request merged**